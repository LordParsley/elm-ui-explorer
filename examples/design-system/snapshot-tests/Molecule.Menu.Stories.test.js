
    const tree = `<div>
    <div class="snapshot" id="Default">
        <div class="navbar-menu" id="navMenubd-example">
            <a class="navbar-item ">
                <span>
                    Home
                </span>
            </a>
            <a class="navbar-item ">
                <span>
                    About
                </span>
            </a>
        </div>
    </div>
</div>`;
    test("Molecule.Menu.Stories", () => {
      expect(tree).toMatchSnapshot();
    });
    