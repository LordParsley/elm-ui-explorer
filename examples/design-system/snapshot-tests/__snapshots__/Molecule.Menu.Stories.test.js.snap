// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Molecule.Menu.Stories 1`] = `
"<div>
    <div class=\\"snapshot\\" id=\\"Default\\">
        <div class=\\"navbar-menu\\" id=\\"navMenubd-example\\">
            <a class=\\"navbar-item \\">
                <span>
                    Home
                </span>
            </a>
            <a class=\\"navbar-item \\">
                <span>
                    About
                </span>
            </a>
        </div>
    </div>
</div>"
`;
